### PR TITLE
[LiveComponent] Move defaultAction validation to compiler pass

### PR DESCRIPTION
**Pull request by norkunas**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Tickets       | N/A
| License       | MIT

Completes `todo` note which was in source code :)

### CODE REVIEW

File: src/LiveComponent/src/DependencyInjection/Compiler/ComponentDefaultActionPass.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * @experimental
+ *
+ * @internal
+ */
+final class ComponentDefaultActionPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container): void
+    {
+        dump($container->findTaggedServiceIds('twig.component'));
+        foreach ($container->findTaggedServiceIds('twig.component') as $class => $component) {
```
Comment (kbond) [inline src/LiveComponent/src/DependencyInjection/Compiler/ComponentDefaultActionPass.php]:
```suggestion
        foreach ($container->findTaggedServiceIds('twig.component') as $class => $component) {
```

**Review by kbond — CHANGES_REQUESTED** [review]
Awesome, thanks for doing this @norkunas! Just a rogue dump that needs to be removed :)

File: src/LiveComponent/src/DependencyInjection/Compiler/ComponentDefaultActionPass.php (hunk 1 above)
Comment (norkunas) [reply src/LiveComponent/src/DependencyInjection/Compiler/ComponentDefaultActionPass.php]: Oops :D

**Review by weaverryan — APPROVED** [review]
👏

**Review by kbond — APPROVED** [review]

**Comment by weaverryan** [comment]
Thanks Tomas!

**Pull request merged**